src/openrct2-ui/ride/Construction.cpp

```diff
@@ -68,15 +89,6 @@ namespace OpenRCT2
         TrackElemType::RightHeartLineRoll,
         kSeparator,
 
-        // Brake for drop
-        TrackElemType::BrakeForDrop,
-        kSeparator,
-
-        // S-Bends
-        TrackElemType::SBendLeft,
```

Comment:
Was moving S-bends intentional? I liked it here.

Reply:
It wasn't intentional, but it wouldn't have made a functional change either whether it's before or after the Tower and the Mini Golf pieces. The relevant tracks would always show them after 'Brake for drop', still, right.

I don't mind either way, so I've just moved them back up.